src/style.css

```diff
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+}
+
+.starsContainer {
+  display: flex;
+  justify-content: center;
+}
+
+.starsField {
+  display: flex;
+  flex-direction: column;
+  width: 122px;
+  height: 96px;
+}
+
+.stars {
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.stars__star {
+  width: 13px;
+  height: 13px;
+  margin: 0 5px 0 2px;
+}
+
+.stars--1 > .stars__star:first-child,
```

Comment:
A lot of repeat code(30-46 rows) try add only
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:not(:nth-child(n + 3)),
.stars--3 .stars__star:not(:nth-child(n + 4)),
.stars--4 .stars__star:not(:nth-child(n + 5)),
.stars--5 .stars__star:not(:nth-child(n+6)) {
  background-image: url(images/star-active.svg);
}
And for star:
 background-image: url(images/star.svg);
  background-repeat: no-repeat;
  background-position: center;